Client/Extensions/UIViewExtensions.swift

```diff
@@ -46,14 +46,6 @@ extension UIView {
         return screenshot(size, offset: offset, quality: quality)
     }
 
-    /*
-     * Performs a deep copy of the view. Does not copy constraints.
-     */
-    @objc func clone() -> UIView {
-        let data = NSKeyedArchiver.archivedData(withRootObject: self)
-        return NSKeyedUnarchiver.unarchiveObject(with: data) as! UIView // swiftlint:disable:this force_cast
```

Comment:
`UIView` does not support `NSSecureCoding` therefore the new APIs are not nice decoding it. However this method isn't actually used anywhere so it's safer to just remove it.